viewing_party/party.py

```diff
@@ -0,0 +1,199 @@
+##Wave 1!!
+
+
+def create_movie(title,genre,rating):
+    
+    if bool(title)==True and bool(genre)==True and bool(rating)==True:
+        dict={"title":"Title A","genre":"Horror","rating":3.5}
+        return dict
+    else:
+        return None
```

Comment:
Even though this works title, genre, and rating are being passed into your function so you do not need to give each argument a value. You can simply do
```suggestion
def create_movie(title, genre, rating):
    if title == None or genre == None or rating == None:
        return None
    else:
        return {
        "title":title, 
        "genre":genre, 
        "rating":rating
        }
```